.github/workflows/reusable-workflow-lint.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint GitHub Actions workflows
+on:
+    workflow_call:
+
+# Disable permissions for all available scopes by default.
+# Any needed permissions should be configured at the job level.
+permissions: {}
+
+jobs:
+    # Runs the actionlint GitHub Action workflow file linter.
+    #
+    # This helps guard against common mistakes including strong type checking for expressions (${{ }}), security checks,
```

Comment:
I assume this also runs the `permissions: {}` top level permissions too?

Reply:
Actionlint only checks that the given permission scopes and access levels are valid -- both at the top level workflow level and the job level -- ie. it lints them but doesn't check for overly broad permissions. [Zizmor does check for excessive permissions](https://woodruffw.github.io/zizmor/audits/#excessive-permissions), but I will be writing a separate proposal for adopting that.